incubator/elasticsearch/README.md

```diff
@@ -56,30 +56,35 @@ $ kubectl delete pvc -l release=my-release,component=data
-| `image.pullPolicy`                   | Container pull policy                   | `Always`                            |
-| `cluster.name`                       | Cluster name          			         | `elasticsearch`                     |
-| `cluster.config`                     | Additional cluster config appended      | `{}`                                |
-| `client.name`                        | Client component name                   | `client`                            |
-| `client.replicas`                    | Client node replicas (deployment)       | `2`                                 |
-| `client.resources`                   | Client node resources requests & limits | `{} - cpu limit must be an integer` |
-| `client.heapSize`                    | Client node heap size                   | `128m`                              |
-| `client.serviceType`                 | Client service type                     | `ClusterIP`                         |
-| `master.name`                        | Master component name                   | `master`                            |
-| `master.replicas`                    | Master node replicas (deployment)       | `2`                                 |
-| `master.resources`                   | Master node resources requests & limits | `{} - cpu limit must be an integer` |
-| `master.heapSize`                    | Master node heap size                   | `128m`                              |
-| `master.name`                        | Data component name                     | `data`                              |
-| `data.replicas`                      | Data node replicas (statefulset)        | `3`                                 |
-| `data.resources`                     | Data node resources requests & limits   | `{} - cpu limit must be an integer` |
-| `data.heapSize`                      | Data node heap size                     | `1536m`                             |
-| `data.storage`                       | Data persistent volume size             | `30Gi`                              |
-| `data.storageClass`                  | Data persistent volume Class            | `nil`                               |
-| `data.terminationGracePeriodSeconds` | Data termination grace period (seconds) | `3600`                              |
-| `data.antiAffinity`                  | Data anti-affinity policy               | `soft`                              |
+|              Parameter               |                             Description                             |               Default                |
+| ------------------------------------ | ------------------------------------------------------------------- | ------------------------------------ |
+| `appVersion`                         | Application Version                                                 | `5.4`                                |
```

Comment:
Please document what the field is used for, so people know when they may have to change it.

Reply:
added a deep dive on application version, https://github.com/kubernetes/charts/pull/1794/files#diff-4c83ed39abf993d7fded2101e2614aecR99